tests/data/cards.json

```diff
@@ -0,0 +1,86 @@
+[
+    {
```

Comment:
what are these data for?

Reply:
This is an example of how the data is returned from the Trello APIs.

I added them as JSON files to use them for mocking so that we don't send real requests to Trello API and face the timeout issue.